yaks-rails/lib/yaks-rails.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails'
+require 'yaks-html'
+
+module Rails
+  module Yaks
```

Comment:
We thought that as well, but we followed the yaks-sinatra implementation where it's also Sinatra::Yaks. So maybe we should consider changing the namespace there too?